frontend/www/js/omegaup/components/arena/ContestListv2.vue

```diff
@@ -348,18 +365,17 @@ export default class ArenaContestList extends Vue {
   @Prop() query!: string;
   @Prop() tab!: ContestTab;
   @Prop() sortOrder!: ContestOrder;
-  @Prop() filterBySignedUp!: boolean;
-  @Prop() filterByRecommended!: boolean;
+  @Prop({ default: () => [] }) filter!: ContestFilter;
```

Comment:
This should be the default value:
```suggestion
  @Prop({ default: ContestFilter.All }) filter!: ContestFilter;
```